doc/source/_static/js/custom.js

```diff
@@ -61,3 +61,23 @@ window.onload = function() {
         localStorage.removeItem("scroll");
     }
 };
+
+
+let firstLink = document.getElementsByClassName("caption")[0];
+firstLink.classList.add("toctree-l1", "current");
+firstLink.style.textTransform = "none";
+firstLink.style.fontWeight = "normal";
+firstLink.innerText = "";
+
+let home = document.createElement("a");
+home.classList.add("reference", "internal");
+
+const version = window.location.href.split("/")[4];
+const res = (version === "latest" || version === "master") ? version : "latest";
+
+home.href = "https://docs.ray.io/en/" + res + "/index.html";
+home.textContent = "Ray Docs Home";
```

Comment:
Can we do this part as part of the toc?

Reply:
@richardliaw as in part of the `toc.yml`? unfortunately, no. doesn't work, because you can't have the same doc twice and `index.html` is already the project root. but since we repurpose the top left logo to redirect to `ray.io` now, we have to resort to tricks like this. :/